Packages/Netherlands3D/WebServiceReader/Runtime/Scripts/WFSReader/WFSInterface.cs

```diff
@@ -37,7 +37,11 @@ void Start()
     //}
     private void BuildFilterInterface(WFSFeature activeFeature)
     {
-        for(int i = filterParent.childCount - 1; i >= 0; i--)
+        print("should build interface for " + activeFeature.FeatureName);
+        return;
```

Comment:
Is deze return nog bewust hier? Want dan kunnen we de code hieronder ook weer lekker opruimen! :)